projects/view/client/activeProjectDetail/activeProjectDetail.html

```diff
@@ -0,0 +1,82 @@
+<!-- refactored too singleProjectDetails.html and rtProjectInfoSection.html and beAPart.html and then merged into rtProjectProfile.html -->
+<template name="activeProjectDetail" class="approvedProjectDetail">
```

Comment:
Does assigning a class here work? As far as I understand, the `<template>` tag does not actually end up in the rendered page. So, any classes or IDs should be assigned to elements inside of the `<template>`.